scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -60,41 +44,152 @@ def test_support_alternative_backends(xp, f_name, n_args, dtype, shapes):
     if is_jax(xp) and f_name == "stdtrit":
         pytest.skip(f"`{f_name}` requires scipy.optimize support for immutable arrays")
```

Comment:
I'll do a second iteration later where this block is absorbed by `xp_capabilities`.
What comes after it is too much of an edge case to be covered by it, so it will remain in the unit test.